content/en/docs/reference/glossary/workload.md

```diff
@@ -1,24 +1,23 @@
 ---
-title: Workloads
+title: Workload
 id: workloads
 date: 2019-02-13
 full_link: /docs/concepts/workloads/
 short_description: >
-   Workloads are objects you use to manage and run your containers on the cluster.
+   A workload is an application running on Kubernetes.
 
 aka: 
 tags:
 - fundamental
-- core-object
 - workload
 ---
-  Workloads are objects you use to manage and run your containers on the cluster.
+   A workload is an application running on Kubernetes.
 
 <!--more--> 
 
-Kubernetes performs the 
-deployment and updates the workload with the current state of the application.
-Workloads include the DaemonSet, Deployments, Jobs, Pods, ReplicaSet, ReplicationController, and StatefulSet objects.
+Various core objects that represent different types or parts of a workload
+include the DaemonSet, Deployment, Job, Pod, ReplicaSet, ReplicationController,
+and StatefulSet objects.
 
 For example, a workload that has a web element and a database element might run the
 database in one {{< glossary_tooltip term_id="StatefulSet" >}} of
```

Comment:
nit: reword: `StatefulSet of Pods`. Is `StatefulSet` good enough?
nit: remove via
One idea:
For example, a workload that has a Web server and a database might run the
database in one {{< glossary_tooltip term_id="StatefulSet" >}} and the Web server
in a {{< glossary_tooltip term_id="Deployment" >}}.

Reply:
great suggestions. done!